src/components/RegionList/RegionListComponent.tsx

```diff
@@ -89,7 +101,7 @@ export class RegionListComponent extends React.Component<WidgetProps> {
         let nameWidth = RegionListComponent.NAME_COLUMN_DEFAULT_WIDTH;
         const availableWidth = this.width - 2 * padding;
         let fixedWidth =
-            RegionListComponent.ACTION_COLUMN_DEFAULT_WIDTH * 2 +
+            RegionListComponent.ACTION_COLUMN_DEFAULT_WIDTH * 3 +
```

Comment:
pleasee help give constants meaningful definition like `OOO_WIDTH`.

Reply:
has changed to `ACTIONS_COLUMN_DEFAULT_WIDTH`.